forgerock-authenticator/src/main/java/org/forgerock/android/auth/PushResponder.java

```diff
@@ -254,18 +254,26 @@ public void onFailure(@NotNull Call call, @NotNull IOException e) {
      *
      * @return OkHttpClient http client
      * */
-    private OkHttpClient getOkHttpClient() {
+    private OkHttpClient getOkHttpClient(URL url) {
         if (httpClient != null) {
             return httpClient;
         }
 
-        httpClient = new OkHttpClient.Builder()
-                .connectTimeout(TIMEOUT, SECONDS)
-                .readTimeout(TIMEOUT, SECONDS)
-                .writeTimeout(TIMEOUT, SECONDS)
-                .followRedirects(false)
+        // Extract host from the URL, it's required to build the NetworkConfig and used by
+        // OkHttpClientProvider. There is no need to monitor the the change of hosts considering
+        // the usage of this SDK.
+        String host = url.getProtocol() + "://" + url.getAuthority();
```

Comment:
You don't need to use the url, I think u can just use url.getAuthority as the host.

Reply:
You are right, I just reused the code from my previous attempt of integration, where the ServerConfig required the 'url' parameter.